### PR TITLE
8/4: Add PerspectiveManager interface & stubbed implementation of PerspectiveManager

### PR DESCRIPTION
**Pull request by annemariecabs**
Add PerspectiveManager interface and the custom exception it throws (NoAppropriateStoryException). Add a stubbed implementation of PerspectiveManager which returns a default PerspectiveDecision instance.

### CODE REVIEW

File: backstory/src/main/java/com/google/sps/perspective/PerspectiveManagerImpl.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+      AttributeType.LIKELY_TO_REJECT,
+      AttributeType.OBSCENE, 
+      AttributeType.PROFANITY, 
+      AttributeType.SEVERE_TOXICITY,
+      AttributeType.SEXUALLY_EXPLICIT, 
+      AttributeType.SPAM, 
+      AttributeType.THREAT,
+      AttributeType.TOXICITY, 
+      AttributeType.UNSUBSTANTIAL
+    };
```
Comment (ykjchen) [inline backstory/src/main/java/com/google/sps/perspective/PerspectiveManagerImpl.java:44]: Nit: looks unrelated to this PR.

Comment (annemariecabs) [reply backstory/src/main/java/com/google/sps/perspective/PerspectiveManagerImpl.java:44]: I just added this because it was important for later, and for the overall implementation of PerspectiveManager interface. I'm hopefully not going to have to make any more changes to the interface.

File: backstory/src/main/java/com/google/sps/perspective/PerspectiveManager.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+ */
+interface PerspectiveManager {
+  /**
+   * Analyzes the passed-in story and returns PerspectiveDecision
+   * containing decision based on analysis.
+   *
+   * @param story The story to be analyzed
+   * @return An object describing the recommendation resulting from the analysis.
+   */
+  public PerspectiveDecision getDecision(String story);
```
Comment (ykjchen) [inline backstory/src/main/java/com/google/sps/perspective/PerspectiveManager.java:29]: @annemariecabs thinking about this more, `get` is often used for property access, so might connote fast access. Maybe consider renaming to `generateDecision()` or something else that indicates that some processing is required?

Comment (annemariecabs) [reply backstory/src/main/java/com/google/sps/perspective/PerspectiveManager.java:29]: Done
